esp-hal/src/i2c/master/mod.rs

```diff
@@ -79,17 +79,33 @@ const MAX_ITERATIONS: u32 = 1_000_000;
 pub enum I2cAddress {
     /// 7-bit address mode type.
     ///
-    /// Note that 7-bit addresses defined by drivers should be specified in
-    /// **right-aligned** form, e.g. in the range `0x00..=0x7F`.
+    /// Note that 7-bit addresses are specified in **right-aligned** form, e.g.
+    /// in the range `0x00..=0x7F`.
     ///
     /// For example, a device that has the seven bit address of `0b011_0010`,
     /// and therefore is addressed on the wire using:
     ///
     /// * `0b0110010_0` or `0x64` for *writes*
     /// * `0b0110010_1` or `0x65` for *reads*
+    ///
+    /// Is specified as 0b0011_0010 or 0x32, NOT 0x64 or 0x65.
```

Comment:
```suggestion
    /// The above address is specified as 0b0011_0010 or 0x32, NOT 0x64 or 0x65.
```

Let's make this a complete sentence